apluslms_roman/cli.py

```diff
@@ -342,20 +400,13 @@ def build_action(context):
     engine = get_engine(context)
     builder = engine.create_builder(config)
 
-    if context.args.list_steps:
-        steps = builder.get_steps()
-        num_len = max(2, len(str(len(steps)-1)))
-        name_len = max(4, len(max(step.name or "" for step in steps)))
-        header_fmt = "{:>%ds}  {:%ds} {}" % (num_len, name_len)
-        step_fmt = "{:%dd}. {:%ds} {}" % (num_len, name_len)
-        print(header_fmt.format('ID', 'NAME', 'IAMGE'))
-        for step in steps:
-            print(step_fmt.format(step.ref, step.name or "", step.img))
-        return
-
     if not verify_engine(engine, only_when_error=True):
         return 1
 
+    # check that _build folder exists, create one if it doesn't
+    if not isdir('_build'):
```

Comment:
If you move this just before https://github.com/apluslms/roman/blob/ea7bcdfdb0133a406bf4d27bbdae6e722b03832d/apluslms_roman/builder.py#L47, then I'll merge it in this commit. Though, add something like `# FIXME: add support for different build folders`.